### DISCUSSION TITLE
Exports a plot to memomy buffer by ggsave

### DISCUSSION THREAD

**Discussion by fmvin** [JetBrains/lets-plot-kotlin#209] [Q&A]
`ggsave` exports plot only to a file. Please extends `ggsave` to export a plot to data buffer.

**Comment by alshan** [comment]
Hi @fmvin , depending on your needs, you can use `PlotSvgExport`, `PlotHtmlExport` or `PlotImageExport` utility, in a way  similar to how they are used in [`ggsave()`](https://github.com/JetBrains/lets-plot-kotlin/blob/master/plot-api/src/jvmMain/kotlin/org/jetbrains/letsPlot/export/ggsave.kt).

Will this work for you? At least as a workaround.

**Comment by alshan** [comment]
Hi, just added small demo which uses `org.jetbrains.letsPlot.core.plot.export.PlotImageExport` utility to obtain bytes of a plot image along with the image dimensions:

https://github.com/alshan/lets-plot-mini-apps/blob/main/jvm-plot-export/src/main/kotlin/ExportPNG.kt
